examples/try_SAR.py

```diff
@@ -0,0 +1,104 @@
+
```

Comment:
There is already a quick example of SAR as a notebook, https://github.com/Microsoft/Recommenders/blob/staging/notebooks/00_quick_start/sar_single_node_movielens.ipynb, which is super similar to this file

Reply:
Yes, it  seems this file can be deleted.

Reply:
Sure I can remove it. 
I just wonder why you prefer a notebook rather than a direct end2end example.

Reply:
It is just the way the repo has been originally designed. We found that there is a high interest from our users to see notebooks instead of pure python files. One reason might be that in a notebook, apart from code you can add formatted comments and images.